nemo/utils/exp_manager.py

```diff
@@ -457,9 +460,11 @@ def get_log_dir(
         name = name or "default"
         version = version or os.environ.get(NEMO_ENV_VARNAME_VERSION, None)
 
-        if version is None:
-            if check_slurm(trainer):
-                logging.warning("Running on a slurm cluster. exp_manager will not add a version number.")
+        if not version:
+            if resume_if_exists:
+                logging.warning(
+                    "No version folders would be created under the log folder as 'resume_if_exists' is enabled."
+                )
```

Comment:
Wouldn't it be more accurate to tell users to use `explicit_log_dir` when resuming, instead of using `exp_dir`/`name`/`version `?